src/math/fan/BigInt.fan

```diff
@@ -82,6 +66,46 @@ native const class BigInt
 	** Decrement by one.  Shortcut is --a or a--.
 	@Operator BigInt decrement()
 
+	////////// mult //////////
+
+	** Multiply this with b.  Shortcut is a*b.
+	@Operator BigInt mult(BigInt b)
+
+	** Multiply this with b.  Shortcut is a*b.
+	@Operator BigInt multInt(Int b)
+
+	////////// div //////////
+
+	** Divide this by b.  Shortcut is a/b.
+	@Operator BigInt div(BigInt b)
+
+	** Divide this by b.  Shortcut is a/b.
+	@Operator BigInt divInt(Int b)
+
+	////////// mod //////////
+
+	** Return remainder of this divided by b.  Shortcut is a%b.
+	@Operator BigInt mod(BigInt b)
+
+	** Return remainder of this divided by b.  Shortcut is a%b.
+	@Operator Int modInt(Int b)
```

Comment:
My thoughts were this:

a%b will always return a number between -b and b.  So there's not a reason to waste a whole BigInt, as the input Int means the return value will **never** be outside Int.minVal & Int.maxVal.  I agree it doesn't look as clean but I figured it would make it more clear as to the range in the return value.

What's your thoughts?  I'm fine changing it but it was just something I thought of.  Personally, I feel like BigInteger.remainder should have taken a BigInteger.  Shame on them :(